elfloader-tool/src/arch-arm/64/crt0.S

```diff
@@ -41,70 +41,6 @@ BEGIN_FUNC(_start)
     b       main
 END_FUNC(_start)
 
-#ifdef CONFIG_IMAGE_BINARY
-BEGIN_FUNC(fixup_image_base)
-    stp     x29, x30, [sp, #-16]!
-    mov     x29, sp
-    ldr     x0, =IMAGE_START_ADDR
```

Comment:
Seem there is no need to have `#include <image_start_addr.h>` in this file any longer then when this usage here is gone.